spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,63 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, some_session
+
+        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'when session has expired' do
+        before do
+          wpcc_expires_at = 5.minutes.ago
+
+          Timecop.travel(Time.current + 30.minutes)
+
+          get :new,
+              nil,
+              age: 34,
+              gender: 'male',
```

Comment:
in fact.. you actually do this further up in another test [Line 108](https://github.com/moneyadviceservice/wpcc/blob/d0baffaece6339c8c9a5a8e8f27fc3ab5adb6922/spec/controllers/your_details_controller_spec.rb#L108)
```
it "'wpcc_expires_at' key should always be updated" do
        session[:wpcc_expires_at] = 5.minutes.ago
        get :new, nil, some_session

        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
      end

```